eng/pipelines/doc-index.yml

```diff
@@ -38,7 +38,6 @@ jobs:
           New-Item -Path "$(Build.SourcesDirectory)/docfx_project" -Name "toc.yml" -Force
           Add-Content -Path "$(Build.SourcesDirectory)/docfx_project/toc.yml" -Value "- name: Azure SDK for JavaScript APIs`r`n  href: api/`r`n  homepage: api/index.md"
           Copy-Item "$(Build.SourcesDirectory)/README.md" -Destination "$(Build.SourcesDirectory)/docfx_project/api/index.md" -Force
-          Copy-Item "$(Build.SourcesDirectory)/README.md" -Destination "$(Build.SourcesDirectory)/docfx_project/index.md" -Force
```

Comment:
I assume this is now being created by the mash-up step?

Reply:
whatever is under /api is moved to / . I had this last statement just as a workaround so we have something on the index.html when we navigate to github.io : `https://karishmaghiya.github.io/azure-sdk-for-js/` But now since we are fixing the landing page issue, I don't need this workaround any more